lib/oidc/handleOAuthResponse.ts

```diff
@@ -83,6 +77,16 @@ export async function handleOAuthResponse(
   // Handling the result from implicit flow or PKCE token exchange
   validateResponse(res, tokenParams);
 
+  if (tokenParams.dpop) {
+    const { allowBearerTokens } = sdk.options?.dpopOptions ?? { allowBearerTokens: false };
```

Comment:
Is there any way to have the default value of `allowBearerTokens` set higher up so we don't need to set it here?

Reply:
Yes and I've done it, but there is no elegant way of telling TS about it without using `sdk.options?.dpopOptions!` (AuthJS is built on a complicated mixin/factory pattern)

Which is preferable?